### PR TITLE
Add e2e tests

### PR DESCRIPTION
**Pull request by ad-m**

### CODE REVIEW

**Comment by fredericosilva** [comment]
Any reason not to use the latest docker-machine https://github.com/docker/machine/releases/tag/v0.16.1 ?

File: e2e/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+RUN curl -s -L "https://github.com/hyperonecom/h1-cli/releases/download/${H1_CLI_VERSION}/h1-alpine" -o /bin/h1 \
+&& chmod +x /bin/h1
+# Build
+WORKDIR /src
+ADD ./ /src/
+RUN go build
+ENV PATH=/src/:$PATH
+
+# Run
+CMD ["docker-machine","create","--help"]
```
Comment (fredericosilva) [inline e2e/Dockerfile]: change to ``` docker-machine create --driver hyperone --help ``` it forces the driver to load

File: e2e/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM golang:alpine
+ENV H1_CLI_VERSION="v1.4.0"
+
+# Setup
+RUN apk add curl bats git make gcc musl-dev findutils grep docker
+RUN base=https://github.com/docker/machine/releases/download/v0.16.0 && \
```
Comment (fredericosilva) [inline e2e/Dockerfile]: Any reason not to use the latest docker-machine https://github.com/docker/machine/releases/tag/v0.16.1 ?

**Comment by ad-m** [comment]
Tests pass.

**Pull request merged**